{"hierarchy":{"paths":[["doc:\/\/GnustoEngine\/documentation\/GnustoEngine","doc:\/\/GnustoEngine\/documentation\/GnustoEngine\/ActionResponse"]]},"variants":[{"traits":[{"interfaceLanguage":"swift"}],"paths":["\/documentation\/gnustoengine\/actionresponse\/internalengineerror(_:)"]}],"sections":[],"identifier":{"url":"doc:\/\/GnustoEngine\/documentation\/GnustoEngine\/ActionResponse\/internalEngineError(_:)","interfaceLanguage":"swift"},"abstract":[{"type":"text","text":"An unexpected internal error occurred within the game engine or an action handler."},{"type":"text","text":" "},{"text":"The ","type":"text"},{"type":"codeVoice","code":"String"},{"type":"text","text":" provides details about the error, primarily for debugging."}],"metadata":{"modules":[{"name":"GnustoEngine"}],"role":"symbol","title":"ActionResponse.internalEngineError(_:)","roleHeading":"Case","fragments":[{"text":"case","kind":"keyword"},{"kind":"text","text":" "},{"kind":"identifier","text":"internalEngineError"},{"text":"(","kind":"text"},{"preciseIdentifier":"s:SS","kind":"typeIdentifier","text":"String"},{"text":")","kind":"text"}],"symbolKind":"case","externalID":"s:12GnustoEngine14ActionResponseO08internalB5ErroryACSScACmF"},"schemaVersion":{"minor":3,"patch":0,"major":0},"primaryContentSections":[{"kind":"declarations","declarations":[{"tokens":[{"text":"case","kind":"keyword"},{"text":" ","kind":"text"},{"kind":"identifier","text":"internalEngineError"},{"text":"(","kind":"text"},{"kind":"typeIdentifier","text":"String","preciseIdentifier":"s:SS"},{"kind":"text","text":")"}],"platforms":["macOS"],"languages":["swift"]}]}],"kind":"symbol","references":{"doc://GnustoEngine/documentation/GnustoEngine/ActionResponse/internalEngineError(_:)":{"role":"symbol","fragments":[{"text":"case","kind":"keyword"},{"text":" ","kind":"text"},{"text":"internalEngineError","kind":"identifier"},{"text":"(","kind":"text"},{"text":"String","kind":"typeIdentifier","preciseIdentifier":"s:SS"},{"text":")","kind":"text"}],"identifier":"doc:\/\/GnustoEngine\/documentation\/GnustoEngine\/ActionResponse\/internalEngineError(_:)","kind":"symbol","title":"ActionResponse.internalEngineError(_:)","url":"\/documentation\/gnustoengine\/actionresponse\/internalengineerror(_:)","type":"topic","abstract":[{"type":"text","text":"An unexpected internal error occurred within the game engine or an action handler."},{"type":"text","text":" "},{"type":"text","text":"The "},{"type":"codeVoice","code":"String"},{"type":"text","text":" provides details about the error, primarily for debugging."}]},"doc://GnustoEngine/documentation/GnustoEngine":{"kind":"symbol","abstract":[],"url":"\/documentation\/gnustoengine","identifier":"doc:\/\/GnustoEngine\/documentation\/GnustoEngine","type":"topic","title":"GnustoEngine","role":"collection"},"doc://GnustoEngine/documentation/GnustoEngine/ActionResponse":{"type":"topic","url":"\/documentation\/gnustoengine\/actionresponse","role":"symbol","fragments":[{"text":"enum","kind":"keyword"},{"text":" ","kind":"text"},{"text":"ActionResponse","kind":"identifier"}],"navigatorTitle":[{"text":"ActionResponse","kind":"identifier"}],"kind":"symbol","identifier":"doc:\/\/GnustoEngine\/documentation\/GnustoEngine\/ActionResponse","abstract":[{"type":"text","text":"Enumerates standard reasons why a game action might fail or be disallowed."}],"title":"ActionResponse"}}}